app/scripts/services/ErrorObject.svc.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+angular.module('yourStlCourts').service('Error', function () {
+  var err = {
+    code:"",
+    message:""
+  }
+  
+  this.errorObject = function(errorCode,errorMessage){
```

Comment:
This is a little non-intuitive code. This looks like a property but is actually a function. I don't believe this service should be holding state at all. Also, I think you're probably looking for a factory, not a service. May I suggest something more like this:

```
angular.module('yourStlCourts').factory('Errors', function ($rootScope) {
  var svc = {
    makeError: makeError,
    ERROR_CODE: ERROR_CODE
  };

$rootScope.$on('$stateChangeError',function(event, toState,toParams,fromState,fromParams,error){
    event.preventDefault();
    $state.go('error', {error: error});
});

 var ERROR_CODE = {
   BAD_REQUEST: 400,
   NOT_FOUND: 404
 }

  function makeError (errorCode, errorMessage) {
    return {
      code: errorCode,
      message: errorMessage
    };
  }

    return svc;
  }
});
```

* "Error" is a keyword in JS so I changed the svc to "Errors"
* Return the service definition via "svc". Only methods on the returned object are exposed to the outside world. This format also makes it easier to unit test.